client/src/components/ListOfItems/style.css

```diff
@@ -0,0 +1,18 @@
+.card {
```

Comment:
Use more descriptive class names. Remember that react is read in node as a bundle, and thus, different classes with the same name might override each other and cause a mess.